resources/dicts/thoughts/alive/apprentice.json

```
@@ -236,28 +348,221 @@
+            "Hides behind r_c during the gathering",
+            "Wonders how r_c can speak with others so easily",
+            "Tries to do some suggestions r_c gave {PRONOUN/m_c/object} to be able to be more outgoing",
+            "Hides away from {PRONOUN/m_c/poss} mentor when {PRONOUN/m_c/subject} {VERB/m_c/learn/learns} {PRONOUN/m_c/poss} mentor wants to take {PRONOUN/m_c/object} to the far edges of the territory today"
+
+        ],
+        "main_trait_constraint": [
+            "insecure",
+            "nervous"
+        ],
+        "random_trait_constraint": [
+            "bold",
+            "daring",
+            "righteous",
+            "adventurous",
+            "flamboyant",
+            "arrogant"
+
+        ],
+        "random_living_status": [
+            "living"
+        ],
+        "relationship_constraint": [
+            "app/mentor"
+        ]
+    },
+    {
+        "id": "insecure_nervous_warrior_apprentice_to_withdrawn_mentor",
+        "thoughts": [
+            "m_c looks at r_c... an awkward silence hangs between them",
+            "m_c feels understood by r_c",
+            "r_c and m_c share prey in comfortable silence",
+            "r_c teaches m_c some techniques {PRONOUN/r_c/subject} {VERB/r_c/use/uses} when being around others gets too much"
+
+        ],
+        "main_trait_constraint": [
+            "insecure",
+            "nervous"
+        ],
+        "random_trait_constraint": [
+            "gloomy",
+            "lonesome",
+            "nervous",
+            "insecure",
+            "grumpy"
+        ],
+        "random_living_status": [
+            "living"
+        ],
+        "relationship_constraint": [
+            "app/mentor"
+        ]
+    },
     {
         "id": "insecure_warrior_apprentice",
         "thoughts": [
-            "Is wondering if {PRONOUN/m_c/subject} {VERB/m_c/are/is} good enough to be a warrior...",
-            "Is reluctant to spar with {PRONOUN/m_c/poss} mentor today",
             "Doesn't think {PRONOUN/m_c/poss} hauls on hunting patrols have been substantial enough",
-            "Wishes {PRONOUN/m_c/subject} could do better",
             "Is sure {PRONOUN/m_c/subject} will fail {PRONOUN/m_c/poss} assessment",
-            "Hopes {PRONOUN/m_c/subject} {VERB/m_c/are/is} not assigned to a border patrol",
-            "Is worried about being an apprentice forever",
+            "Thinks that {PRONOUN/m_c/subject}'ll mess up on a border patrol",
             "Isn't sure if anyone would trust {PRONOUN/m_c/object} to do a good job",
-            "Is fighting back tears after missing a catch",
-            "Is worried to lose in a real fight",
             "Is worrying about something",
             "Says sorry more often than {PRONOUN/m_c/subject} need to",
             "Is hiding after receiving a compliment",
-            "Is avoiding everyone today"
+            "Has had a good day, and actually feels confident in {PRONOUN/m_c/self} today"
```

```suggestion
            "Has had a good day, and actually feels confident in {PRONOUN/m_c/self}"
```

---

The today implies this isn't normal, hinting at it being an insecure thought /info

---

The "actually" does that already - "actually" implies that it's abnormal/unusual to happen, acting as an intensifier in this context /info
Otherwise:
```suggestion
            "Has had a good day today, and actually feels confident in {PRONOUN/m_c/self}"
```

---

Alright then, makes sense.